stdlib/LinearAlgebra/src/lq.jl

```diff
@@ -67,15 +67,15 @@ LQPackedQ(factors::AbstractMatrix{T}, τ::Vector{T}) where {T} = LQPackedQ{T,typ
 """
     lq!(A) -> LQ
 
-Compute the LQ factorization of `A`, using the input
+    Compute the [`LQ`](@ref) factorization of `A`, using the input
```

Comment:
The 4-spaces shift is probably an accident.

Reply:
Thanks!